Resources/skeleton/app/SymfonyRequirements.php

```diff
@@ -387,13 +387,21 @@ public function __construct()
 
         $installedPhpVersion = phpversion();
 
+        $composerFileContents = file_get_contents(__DIR__.'/../composer.json');
+        if (array_key_exists('php', $composerFileContents['require'])) {
+            $phpDependency = $composerFileContents['require']['php'];
```

Comment:
@stof suggested that the required PHP version depends on the required symfony version. You're using the required PHP version here. Not sure what makes more sense.